composer.json

```
@@ -30,7 +29,8 @@
         }
     },
     "suggest": {
-        "ext-pdo_sqlite": "For running functional tests out-of-the-box this is required"
+        "ext-pdo_sqlite": "For running functional tests out-of-the-box this is required",
+        "neos/fluid-adaptor": "For rendering HTML templates"
```

```suggestion
        "neos/fluid-adaptor": "For rendering templates with TYPO3.Fluid"
```
Fluid can render more than HTML ;)

---

I will copy the same description to flow-development-collection :)